pkgs/development/python-modules/zebrafy/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  pillow,
+  pypdfium2,
+  pytest-cov,
+}:
+
+buildPythonPackage rec {
+  pname = "zebrafy";
+  version = "1.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "miikanissi";
+    repo = "zebrafy";
+    tag = version;
+    hash = "sha256-B8jrFQh5swDMfYjdMcY0Hh2VAzknDwarDKVAML6F2r4=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    setuptools-scm
+    pillow
+    pypdfium2
+  ];
+
+  pythonImportsCheck = [ "zebrafy" ];
+
+  nativeCheckInputs = [
+    pytest-cov
```

Comment:
```suggestion
    pytest-cov-stub
    pytestCheckHook
```

Reply:
Why the stub?

Reply:
We don't want to test coverage, but tests will fail without it.